state_city.js

```diff
@@ -0,0 +1,43 @@
+
+const data = { 
+    "india": {
+      "gujarat": ["Surat", "Rajkot", "Jamnagar", "Vadodara"],
+      "up": ["Kanpur", "Agra", "Noida", "Saharanpur"],
+      "dehli": ["Gokalpuri", "New-Dehli", "Mandoli", "Taj-Pul"]
+    },
+    "usa": { 
+      "texas": ["Austin", "SanAntonio", "Dallas", "Houston"],
+      "chicago": ["Lyons", "ForestPark", "RiverForest", "Lincolnwood"],
+      "lasVegas": ["SpringValley", "Boulder", "Paradise", "Henderson"]
+    }
+  };
+function getCountryValue(){
+    let countryDropdown = document.getElementById("country").value;
+    var stateDropdown = document.getElementById("states");
```

Comment:
I think, give variable names in snake case like `state_dropdown` and `country_dropdown`. This way we can better distinguish between function name and variable name.